annotation/tests/test_distribution.py

```diff
@@ -1277,3 +1284,48 @@ def test_get_page_number_combinations(
     expected: List[int],
 ) -> None:
     assert get_page_number_combinations(file_pages, user_pages) == expected
+
+
+@pytest.mark.parametrize(
+    (
+        "split_multipage_doc_setup",
+        "validation_type",
+        "annotators",
+        "validators",
+        "annotation_tasks",
+        "expected_output",
+    ),
+    [
+        (
+            "",
+            ValidationSchema.cross,
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [{"user_id": "2"}],
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [],
+        ),
+        (
+            "1",
+            ValidationSchema.hierarchical,
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [{"user_id": "2"}],
+            [{"user_id": "1"}],
+            [{"user_id": "2"}],
+        ),
+    ],
+)
+def test_choose_validators_users(
+    split_multipage_doc_setup: str,
+    validation_type: ValidationSchema,
+    annotators: List[User],
+    validators: List[User],
+    annotation_tasks: List[Task],
+    expected_output: List[User],
+):
+    with patch.dict(
+        "os.environ", {"SPLIT_MULTIPAGE_DOC": split_multipage_doc_setup}
```

Comment:
Patching os.environ is not the best practice. Patching env vars overall. Let's patch annotation.distribution.main.SPLIT_MULTIPAGE_DOC value itself